### PR TITLE
Create 2024-04-17.md ACA coverage

### PR DESCRIPTION
**Pull request by camronmartinez**

### CODE REVIEW

File: content/verification/veteran-verification/release-notes/2024-04-17.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+We added a new endpoint: ACA Coverage (restricted access) for v2 of the Veteran Service History and Eligibility API.
+* Authorized users can send requests to this endpoint to check Veteran enrollment in minimum essential coverage (MEC) as defined by the Affordable Care Act (ACA). This enrollment inquiry is for a specified date range in requests (required).
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-04-17.md:2]: Does the required mean that a user has to submit a specified date range?

**Review by joeabraun — COMMENTED** [review]
Just a question!

File: content/verification/veteran-verification/release-notes/2024-04-17.md (hunk 1 above)
Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-04-17.md:2]: Yes, it is required that the user submit a specific date range

Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-04-17.md:2]:
```suggestion
* Authorized users can send requests to this endpoint to check a Veteran's enrollment in minimum essential coverage (MEC) as defined by the Affordable Care Act (ACA). This enrollment request requires a specified date range. 
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
One last change

**Review by joeabraun — APPROVED** [review]
UX approves

**Pull request merged**